src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -556,6 +567,7 @@ static void uct_ib_mlx5_devx_reg_symmetric(uct_ib_mlx5_md_t *md,
     uint32_t symmetric_rkey;
     ucs_status_t status;
 
+    ucs_assert(!(memh->super.flags & UCT_IB_MEM_MULTITHREADED));
```

Comment:
@tvegas1 do we need to update symmetric key creation? To check whether original memh was registered in multiple threads or not

Reply:
if ksm (symmetric key) in front of ksm (multi-thread) is not supported, i think we should either:
1. prevent MT when `((flags & UCT_MD_MEM_SYMMETRIC_RKEY) && (md->flags & UCT_IB_MLX5_MD_FLAG_MKEY_BY_NAME_RESERVE)` is true
2. prevent symmetric key when multi-thread flag is set on the memh

Reply:
started #9447 to skip altogether mt reg for smkey.